<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.placepass.marriottMoments.tests.TestClass1" tests="9" time="465.999" ignored="0" failures="1" timestamp="13 Aug 2018 14:30:22 GMT" skipped="0" hostname="LAPTOP-Q794HR9O" errors="0">
  <testcase name="tc4_verifyAavilabityDates" time="101.322" classname="com.placepass.marriottMoments.tests.TestClass1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.placepass.marriottMoments.pages.BookPage.verifyAvailability(BookPage.java:65)
	at com.placepass.marriottMoments.tests.TestClass1.tc4_verifyAavilabityDates(TestClass1.java:56)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- tc4_verifyAavilabityDates -->
  <testcase name="tc6_BookCalenderShouldApper" time="71.194" classname="com.placepass.marriottMoments.tests.TestClass1"/>
  <testcase name="tc2_verifyDestinationSearch" time="24.080" classname="com.placepass.marriottMoments.tests.TestClass1"/>
  <testcase name="tc7_BookWith1Guest" time="99.190" classname="com.placepass.marriottMoments.tests.TestClass1"/>
  <testcase name="tc0_verifyAllLinks" time="17.687" classname="com.placepass.marriottMoments.tests.TestClass1"/>
  <testcase name="tc3_verifyDestinationSearch" time="21.425" classname="com.placepass.marriottMoments.tests.TestClass1"/>
  <testcase name="tc5_maxPriceValidation" time="21.714" classname="com.placepass.marriottMoments.tests.TestClass1"/>
  <testcase name="tc7_BookWithOutPayentInfo" time="98.910" classname="com.placepass.marriottMoments.tests.TestClass1"/>
  <testcase name="tc1_verifyDestinationPageNavigation" time="10.477" classname="com.placepass.marriottMoments.tests.TestClass1"/>
</testsuite> <!-- com.placepass.marriottMoments.tests.TestClass1 -->
